backend/src/src-cvesearch/src/test/resources/sw360.properties

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) Bosch Software Innovations GmbH 2016.
+#
+# All rights reserved. This configuration file is provided to you under the
+# terms and conditions of the Eclipse Distribution License v1.0 which
+# accompanies this distribution, and is available at
+# http://www.eclipse.org/org/documents/edl-v10.php
+
+cvesearch.default.vendor.threshold=1
+cvesearch.default.product.threshold=0
+cvesearch.default.cutoff=6
```

Comment:
missing newline at EOF

Reply:
Doesn't it count as formatting? I fixed this anyway :)

Reply:
Yes, it is formatting, obviously. But that does not count as changing someone else's formatting.

Reply:
done.